web/src/app/WebApp/Installers/Joomla/JoomlaSetup.php

```diff
@@ -95,7 +96,7 @@ public function install(array $options = null): bool {
 			"--db-pass=" . $dbPass,
 			"--db-name=" . $dbName,
 			"--db-prefix=" . Util::generate_string(5, false) . "_",
-			"--db-host=localhost",
+			"--db-host=" . $dbHost,
```

Comment:
this should be
```php
"--db-host=".\Hestiacp\quoteshellarg\quoteshellarg($dbHost);
```
but we do the same mistake 10 times above, so you don't need to fix it in this PR if you don't want to

Reply:
Thanks for your comments! Indeed, I basically copied the existing style. I think it makes sense to update all of those instances separately.

Reply:
@etfz agreed :)

Reply:
fixed in https://github.com/hestiacp/hestiacp/pull/4563